tests/test_configurations/test_manager.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""This test module contains the tests for the configurations manager module."""
+
+import os
+from copy import deepcopy
+from unittest.mock import patch
+
+import pytest
+import yaml
+
+from aea.configurations.manager import AgentConfigManager
+
+
+agent_config_data = yaml.safe_load(
+    """
```

Comment:
move into data as a file?

Reply:
it used in one place only, it's easier to see it in file